frontend/javascripts/dashboard/dataset/queries.tsx

```diff
@@ -324,6 +324,7 @@ export function useUpdateFolderMutation() {
               ? {
                   ...updatedFolder,
                   parent: oldFolder.parent,
+                  metadata: oldFolder.metadata,
```

Comment:
I wonder whether these two lines should be above `...updatedFolder` so that they don't have precedence. however, moving a folder and updating the metadata seem to work currently :thinking: